source/models/defining-models.md

```diff
@@ -123,7 +132,10 @@ attribute types, and new types can be registered as transforms. See the
 ### Defining Relationships
 
 Ember Data includes several built-in relationship types to help you
-define how your models relate to each other.
+define how your models relate to each other. Ember Data has two
```

Comment:
This paragraph talks both about relationship types and relationship methods, I think thats a bit too much/confusing. Maybe we can collapse the whole paragraph into fewer sentences that say, hey, you can declare relationships using 'belongsTo' and 'hasMany'.